### PR TITLE
Add get share link to hopeful

### PR DESCRIPTION
**Pull request by LibertyLocked**
Allows user to generate a share link for a hopeful

### CODE REVIEW

**Comment by modulebaan** [comment]
As far as I know, this is deprecated in the official app. But it may be usefull for someone.

File: pynder/models/user.py [hunk 1]
```diff
@@ -117,6 +117,9 @@ def superlike(self):
     def dislike(self):
         return self._session._api.dislike(self.id)
```
Comment (charliewolf) [inline pynder/models/user.py]: I'd like to see this as a `@property`

File: pynder/models/user.py [hunk 2]
```diff
@@ -117,6 +117,9 @@ def superlike(self):
...
 
+    def share(self):
```
Comment (charliewolf) [inline pynder/models/user.py]: Maybe change this to `share_link`

**Comment by LibertyLocked** [comment]
I've added requested changes.

**Pull request merged**